instance_config.json

```diff
@@ -1,3 +0,0 @@
-{
-  "conduit_token": "api-6fd64u2sr4xk27gazrttzzuxoyke"
```

Comment:
Not sure what is the best approach here, but this file is referenced in the top-level docker-compose.yml, so it should at least exist with instructions for how to populate.

Reply:
Ideally we should aim to remove this, I've left it for now since I am not sure if web/worker will break if I just remove the volume mount.